CONTRIBUTING.rst

```diff
@@ -1,7 +1,10 @@
 Contributing
 =========================
 
-As an open source project, Mesa welcomes contributions of many forms.
+As an open source project, Mesa welcomes contributions of many forms, and from beginners to experts. If you are
```

Comment:
Minor nit: this should be "open-source project" (see https://en.wikipedia.org/wiki/Open-source_software). A lot of people don't put the hyphen anyway. Can be fixed in a separate PR, because I'm sure I have seen the phrase without the hyphen elsewhere in the repo.
Note that "Firefox is an open-source software" and "Firefox is open source" are both correct.